crates/biome_cli/src/execute/traverse.rs

```diff
@@ -553,14 +553,13 @@ impl TraversalOptions<'_, '_> {
 }
 
 /// Path entries that we want to ignore during the OS traversal.
-pub const IGNORE_ENTRIES: &[&[u8]] = &[
-    b".cache",
+pub const TRAVERSAL_IGNORE_ENTRIES: &[&[u8]] = &[
     b".git",
     b".hg",
     b".svn",
     b".yarn",
-    b".timestamp",
```

Comment:
Why is `.timestamp` removed here? I think it makes sense for `TRAVERSAL_IGNORE_ENTRIES` to be a superset of `SCANNER_IGNORE_ENTRIES`.

Reply:
I am not very comfortable about this change, I think we should discuss it. For example, ignoring without documentation the `.vercel` or `.next` folders should be documented and explained.